ui/app/models/pki/pki-role-engine.js

```diff
@@ -117,6 +165,8 @@ export default class PkiRoleEngineModel extends Model {
           ],
         },
       ]);
+      // to modify allowedDomains
+      // console.log((this._fieldToAttrsGroups[1]['Domain handling'][0].type = 'string'), 'here!!!!');
```

Comment:
keeping her for future reference on how to modify the internals of these properties coming from openAPI. Will be helpful if I need to change a tooltip, etc.